src/openzaak/components/catalogi/admin/utils.py

```diff
@@ -219,6 +226,19 @@ def import_zaaktype_for_catalogus(
                                 "A validation error occurred while deserializing a {}\n{}"
                             ).format(resource, deserialized.errors)
                         )
+            else:
+                files_not_found.append(f"{resource}.json")
+
+    if len(files_found) < 1:
+        msg = _(
+            "No files found. Expected: {files_not_found} but received:<br> {files_received}"
+        )
+        msg_dict = {
+            "files_not_found": ", ".join(files_not_found),
+            "files_received": ", ".join(files_received),
+        }
+
+        raise ValueError(format_html(msg, **msg_dict))
```

Comment:
I think `CommandError` is a better fit here, because you don't need to write a separate handler for it. Now you don't catch ValueError in `CatalogusZaakTypeImportSelectView`. Is it not possible to be raised there?

Reply:
I replaced `ValueError` with `CommandError`.

It can be raised in `CatalogusZaakTypeImportSelectView` but it would be weird for `InformatieObjectType.json` or `BesluitType.json` are in the correct directory but the other files are not.

Reply:
Also based on the second comment, it is possible that some of the files are in the base zip directory and some files are in a subdirectory, which will not throw the error. Probably will still fail, but how likely is this to happen?

Reply:
I have tested with the `CatalogusZaakTypeImportSelectView` and it works as expected